handler/user.py

```diff
@@ -21,6 +22,32 @@ def verify_params(params, required_params):
 
 class UserHandler(BaseHandler):
 
+    @staticmethod
+    def do_login(json):
+        try:
+            user = User.do_login(json)
+            password = json['password']
+            if user and bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
```

Comment:
Isn't the `user.password` already encoded? Or is that how the method `checkpw` works?

Reply:
That is how the method works. If I removed the encode part it doesn't work.  There is a Server error.

Reply:
Alright, well everything looks good. Nice job!